src/Paprika/Merkle/ComputeMerkleBehavior.cs

```diff
@@ -649,7 +649,9 @@ private void EncodeBranch(scoped in Key key, scoped in ComputeContext ctx, scope
         // Write length of length in front of the payload, resetting the stream properly
         var end = stream.Position;
         var actualLength = end - initialShift;
-        var lengthOfLength = Rlp.LengthOfLength(actualLength) + 1;
+        var lengthOfLength = Rlp.LengthOfLength(actualLength);
+        if (actualLength >= 56) //to match StartSequence
```

Comment:
Should we have it extracted as a const under `Rlp` maybe? Then both `Rlp.LengthOfLength` and this place could refer to it properly?

Reply:
Added constant in `Rlp`